### PR TITLE
media-libs/libopenshot-audio: add missing DEPEND on x11-libs/libXcursor

### PR DESCRIPTION
**Pull request by sbraz**
Fixes https://bugs.gentoo.org/594664

### CODE REVIEW

File: media-libs/libopenshot-audio/libopenshot-audio-0.1.2.ebuild [hunk 1]
```diff
@@ -21,6 +21,7 @@ RDEPEND="
 	x11-libs/libXext
 "
 DEPEND="
+	x11-libs/libXcursor
```
Comment (mgorny) [inline media-libs/libopenshot-audio/libopenshot-audio-0.1.2.ebuild]: Hmm, why does it have build-time-only deps on those libraries? Doesn't it want some of -proto packages instead?

**Comment by sbraz** [comment]
Fixed now :)

**Comment by mgorny** [comment]
Doesn't the same (dlopen) apply to Xinerama and the other lib as well?

**Comment by sbraz** [comment]
Well yes, they're now in RDEPEND too

**Comment by mgorny** [comment]
Right, sorry, read the diff wrong.

**Comment by SoapGentoo** [comment]
@sbraz you've changed `RDEPEND`, requires a revbump

**Pull request closed without merging**